src/console/index.tsx

```diff
@@ -5,6 +5,10 @@ import { AppProvider } from "./app/provider";
 import App from "./App";
 import "./index.css";
 
+const colorScheme = (new URLSearchParams(window.location.search)).get('colorScheme')
+/* @ts-ignore */
+window.document.documentElement.style.colorScheme = colorScheme
```

Comment:
The `colorScheme` property is not on the `CSSStyleDeclaration` type in the project's TypeScript version, which is currently pinned to `4.3.5`.

We can remove the `@ts-ignore` by upgrading to TypeScript 4.4+, but I wasn't sure if there was a specific reason that it was pinned to `4.3.5`.